ztp/source-crs/extra-manifest/disk-encryption.yaml.tmpl

```diff
@@ -34,13 +54,15 @@ spec:
             [Unit]
             Description=Sync the main system network configuration into initramfs for consistency at boot
             After=NetworkManager.service
+            ConditionKernelCommandLine=ignition.firstboot
 
             [Service]
             Type=simple
-            ExecStart=rpm-ostree ex initramfs-etc --track /etc/NetworkManager
+            ExecStart=/bin/bash -c "/bin/rpm-ostree initramfs-etc --track /etc/NetworkManager && /bin/rpm-ostree status && /bin/systemctl reboot"
```

Comment:
can you please move this to a separate commit that would be about fixing NBDE?

Reply:
fixed